zScanner/Views/Departments/ErrorView.swift

```diff
@@ -39,9 +39,9 @@ class ErrorView: UIView {
         
         addSubview(reloadButton)
         reloadButton.snp.makeConstraints { make in
-            make.top.equalTo(messageLabel.snp.bottom).offset(8)
-            make.trailing.leading.equalToSuperview().inset(64)
-            make.bottom.equalToSuperview()
+            make.top.equalTo(messageLabel.snp.bottom).offset(16)
+            make.trailing.leading.equalToSuperview().inset(120)
```

Comment:
Inset 120 doesn't seem correct. If you want a smaller button, setup `centerY` and `width` instead.

Reply:
Yep, what I was talking about, this is how the button looks on iPhone SE

![IMG_53604A7CEF8F-1](https://user-images.githubusercontent.com/7560049/75717744-a682e700-5cc9-11ea-9dd1-7af6fcc3eadb.jpeg)

Reply:
Better to set `contentEdgeInsets` on the button.